Nether Vein Changes & Additions

voidsong-dragonfly opened the pull request:
- New nether veins of various types!
     - One for new soulsand based vein, including soul soil, soulsand, and gravel
     - One for magma-related blocks, including magma blocks, smooth basalt, and obsidian
 - The Archaic Digsite vein now has a more-fitting blockset
     - Removed gravel, instead added polished blackstone & bricks
     - Added gilded blackstone to the vein at the lowest priority

voidsong-dragonfly committed:
Change Ancient Debris to be blackstone

voidsong-dragonfly committed:
Add in Soul Silt Vein

voidsong-dragonfly committed:
Add in Cooled Lava Tube vein

BluSunrize commented:
This needs an update with spoils now merged

BluSunrize commented:
Oh also, the Silt vein might need an overhaul, since gravel is a spoil now?
Alternatively, should we maybe not have spoil on silt?

voidsong-dragonfly commented:
My thought is something like having dirt replace it, maybe? Sand + Clay + Dirt instead of gravel? Makes the vein less directly-useful for concrete, though.

voidsong-dragonfly committed:
Final fixes before un-draft

 - Add nether spoils to nether veins I forgot
 - Change silt spoils to be coarse dirt instead of gravel

BluSunrize commented on src/datagen/java/blusunrize/immersiveengineering/data/Recipes.java:
```diff
@@ -1232,12 +1232,32 @@ private void mineralMixes(@Nonnull Consumer<FinishedRecipe> out)
+				.addOre(Blocks.POLISHED_BLACKSTONE_BRICKS, .4f)
+				.addOre(Blocks.POLISHED_BLACKSTONE, .3f)
+				.addOre(Blocks.ANCIENT_DEBRIS, .2f)
+				.addOre(Blocks.GILDED_BLACKSTONE, .1f)
```
I don't love the reduction in Ancient Debris output here.
Is there a reason other than "making space for everything else"? I must admit, I have not *played* with it, so maybe it produces ridiculous amounts of Ancient Debris?

voidsong-dragonfly replied:
Because, when I was playing around, the ancient debris haul was still _huge_ - that 0.5 instead of 0.7 on fail chance actually means it's closer to 0.7x instead of 0.5x the amount of debris. It was still a huge amount for me, especially compared to the rarity of debris in the nether.

The below image is two minutes (approximate) of runtime on an archaic digsite of 99.99% saturation
![image](https://user-images.githubusercontent.com/40431605/187919919-59f074b9-aac1-4b58-92ae-488f47412074.png)

BluSunrize reviewed (approved):
LGTM, but one question on balancing
